0000-template.md

```diff
@@ -0,0 +1,103 @@
+- Feature Name: (fill me in with a unique identifier, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Grid Issue: (leave this empty)
```

Comment:
Use the full, official name "Hyperledger Grid".

- My recommendation: s/Grid/Hyperledger Grid/g

- Or say "Hyperledger Grid" the first time it's used (line 4)  a

Reply:
I changed it in the header (since that text will be in the final RFC output), but not later in the document since this document is not introducing Grid and thus the informal use is clearer (because it is brief).